hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseTables.java

```diff
@@ -143,19 +161,28 @@ public static Edge out(String store) {
             return new Edge(store, true);
         }
 
+        public static Edge out(String store, HugeConfig config) {
+            return new Edge(store, true, config);
+        }
+
+        public static Edge in(String store, HugeConfig config) {
+            return new Edge(store, false, config);
+        }
+
         public static Edge in(String store) {
             return new Edge(store, false);
         }
 
         @Override
         public void insert(Session session, BackendEntry entry) {
-            long ttl = entry.ttl();
+            BackendEntry tmpEntry = hbaseSerializer.serializerForEdgeEntry(entry);
```

Comment:
we expect the entry is serialized by the upper layer, and don't need to handle it anymore here. can we move it to Serializer layer? and don't hold a hbaseSerializer instance in the table class.

Reply:
A current solution: modify the vertex-edge deserialization method in HbaseSerializer to the BinarySerializer class

Reply:
I think it's OK

Reply:
I encountered a problem during the transformation:
When the byte obtained from HBase is deserialized into BinaryBackendEntry, an exception occurs when the first two bytes are added to the pre-partition parsing Vertex and Edge

 `HbaseTable.java 
entry = new BinaryBackendEntry(type, id)
                             |
                             |
                            V
BinaryBackendEntry.java
public BinaryBackendEntry(HugeType type, byte[] bytes) {
        this(type, BytesBuffer.wrap(bytes).parseId(type))
    }
                             |
                             |
                            V
public BinaryId parseId(HugeType type) {
         if (type.isIndex()) {
             return this.readIndexId(type);
         }
         // Parse id from bytes
         int start = this.buffer.position();
         /*
          * Since edge id in edges table doesn't prefix with leading 0x7e,
          * so readId() will return the source vertex id instead of edge id,
          * can't call: type.isEdge() ? this.readEdgeId() : this.readId();
          */
         Id id = this.readId(); **_//an exception occurs when the first two digits are added to the pre-partition parsing Vertex and Edge_**
         int end = this.buffer.position();
         int len = end - start;
         byte[] bytes = new byte[len];
         System.arraycopy(this.array(), start, bytes, 0, len);
         return new BinaryId(bytes, id);
}`

Ask if you have any suggestions @javeme @imbajin

Reply:
We can keep a symmetrical write/read process.

tips for minor improvement of github comment style: we can use triple backticks to include code blocks, like:
```java
    ```java
    java code put here
    ```
```